configuration header (#112)
ComputationalRadiationPhysics/cupla: README.md

- Checkout the [tuning guide](doc/TuningGuide.md) for a step further to performance
  portable code.

> [cupla can be used with single standalone header](doc/StandaloneHeader.md)

Reviewer: ```suggestion
[cupla can be used as a standalone header-only library](doc/StandaloneHeader.md)
```
